kiali.go

```diff
@@ -73,7 +74,7 @@ func main() {
 	validateFlags()
 
 	// log startup information
-	log.Infof("Kiali: Version: %v, Commit: %v\n", version, commitHash)
+	log.Infof("Kiali: Version: %v, Commit: %v, Go: %v", version, commitHash, goVersion)
```

Comment:
I assume losing the newline is intentional, as I think Infof already adds a newline.

Reply:
Yes. With the `\n` it actually produces a blank line after that line, which had no purpose and made the log look like something was missing. I took it out because it wasn't needed, Infof logs a newline at the end already